docs/getting-started-v2.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: page-with-side-nav
+title: Getting Started
+---
+# Getting Started
+
+De '{{site.apiname}}' Web API is gespecificeerd in OpenAPI specifications (OAS).
+
+Wil je de API gebruiken? Dit kun je doen:
+
+1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
+2. [Implementeer de API Client](#implementeer-de-api-client)
+3. [Probeer en test de API](#probeer-en-test-de-brpproxy)
+
+## Functionaliteit en specificaties
+
+Je kunt een visuele representatie van de specificatie genereren met [Redoc]({{ site.baseurl }}/redoc-io).
+
+De (resolved) OAS3 kun je hier downloaden: [openapi.yaml](https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/develop/specificatie/genereervariant/openapi.yaml).
+
+De [functionele documentatie](./features-v2) vind je in [features](./features-v2).
+
+## Implementeer de API client
+Client code kun je genereren met de "[genereervariant](https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/develop/specificatie/genereervariant/openapi.yaml){:target="_blank" rel="noopener"}" van de API-specificaties en een code generator. Een overzicht met codegeneratoren kun je vinden op [OpenAPI.Tools](https://openapi.tools/#sdk){:target="_blank" rel="noopener"}.
+
+Deze repo bevat scripts waarmee je met [OpenAPI Generator](https://openapi-generator.tech/){:target="_blank" rel="noopener"} client code kunt genereren in JAVA, .NET (Full Framework & Core) en Python. De makkelijkste manier om de code generatie scripts te gebruiken, is door deze repo te clonen. Na het clonen kun je met `npm install` de benodigde packages installeren en kun je met `npm run <script naam>` één van de volgende scripts uitvoeren:
+- oas:generate-java-client (voor JAVA client code)
+- oas:generate-netcore-client (voor .NET Core client code)
+- oas:generate-net-client (voor .NET Full Framework client code)
+- oas:generate-python-client (voor Python client code)
+
+Een lijst met andere ondersteunde generator opties kun je vinden in de [Generators List](https://openapi-generator.tech/docs/generators){:target="_blank" rel="noopener"} van OpenAPI Generator.
+
+Note. De prerequisite van OpenAPI Generator is JAVA. Je moet een JAVA runtime installeren voordat je OpenAPI Generator kunt gebruiken
+
+## Probeer en test de BrpProxy
+
+Door wettelijke restricties kan de BRP bevragen API geen informatie vragen leveren. Er wordt op dit moment aan gewerkt om deze restricties weg te halen. Totdat dit is gerealiseerd moet de BrpProxy worden gebruikt om de informatie vragen te kunnen krijgen.
+
+De BrpProxy is geïmplementeerd als een containerized applicatie, zodat het makkelijk kan worden getest op een lokale machine en worden gehost in een productie omgeving.
```

Comment:
De BrpProxy is geïmplementeerd als een containerized applicatie, zodat het makkelijk kan worden getest op een lokale machine en gehost in een productie omgeving.